integrations/lib/testing/integration/authservice.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/gravitational/teleport/integrations/lib/logger"
 )
 
-var regexpAuthStarting = regexp.MustCompile(`Auth service [^ ]+ is starting on [^ ]+:(\d+)`)
+var regexpAuthStarting = regexp.MustCompile(`listen_address:[^ ]+:(\d+)`)
```

Comment:
should this one also have like `Auth service is starting....`?

Reply:
Do you think it's worth making sure the listen_address field is a constant, and/or adding a comment to the corresponding log statement that those messages shouldn't change?

Reply:
This is a work in progress. Those tests running the teleport binary as they are fragile and slow, and it's hard to attach a debugger. The access plugins tests are now starting a teleport instance in-process. Terraform and event-exporter tests are still using this test suite.

This will be removed once we merge teleport-plugins with teleport.